.github/workflows/agw-build-publish-container.yml

```diff
@@ -188,3 +188,69 @@ jobs:
       registry: ${{ needs.build-containers.outputs.registry }}
       test_targets: extended_tests
     secrets: inherit
+
+  publish-container-tests:
+    runs-on: ubuntu-20.04
+    if: always() && github.event_name == 'push'
+    needs: [test-containers-precommit, test-containers-extended]
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # pin@v3.1.0
+
+      - name: Collect all test results
+        run: |
+          mkdir -p lte/gateway/test_results
+          if [ -n ${{ needs.test-containers-precommit.outputs.final_status }} ]
```

Comment:
Please add quotes around the variable, else the check may fail if you have `if [ -n ]` without any string at all.

Reply:
It is safer to quote all variables in general to avoid unintended effects if strings are empty or contain spaces.

Reply:
Changed.